contract_function_result.go

```diff
@@ -232,6 +236,35 @@ func (result ContractFunctionResult) GetInt256(index uint64) []byte {
 	return result.ContractCallResult[index*32 : index*32+32]
 }
 
+func toBigIntFromTwosComplement(data []byte) *big.Int {
```

Comment:
Here again I think we can use https://pkg.go.dev/github.com/ethereum/go-ethereum/common/math#S256